index.ts

```diff
@@ -0,0 +1,81 @@
+import { NativeModules } from 'react-native'
+
+const { RNZoomUs } = NativeModules
+
+if (!RNZoomUs) console.error('RNZoomUs native module is not linked.')
+
+async function initialize(
+  params: { clientKey: string; clientSecret: string; domain?: string },
+  settings?: {
+    // ios only
+    disableShowVideoPreviewWhenJoinMeeting?: boolean
+  },
+) {
+  if (typeof params !== 'object') {
+    throw new Error(
+      'ZoomUs.initialize expect object param. Consider to check migration docs. ' +
+        'Check Link: https://github.com/mieszko4/react-native-zoom-us/blob/master/docs/UPGRADING.md',
+    )
+  }
+
+  if (!params.domain) params.domain = 'zoom.us'
+
+  return RNZoomUs.initialize(params, {
+    // more details inside: https://github.com/mieszko4/react-native-zoom-us/issues/28
+    disableShowVideoPreviewWhenJoinMeeting: true,
+    ...settings,
+  })
+}
+
+async function joinMeeting(params: {
+  userName: string
+  meetingNumber: string | number
+  password?: string
+  participantID?: string
+  noAudio?: boolean
+  noVideo?: boolean
+
+  // IOS only fields:
+  zak?: string
+  webinarToken?: string
+}) {
+  let { meetingNumber, noAudio = false, noVideo = false } = params
+  if (!meetingNumber) throw new Error('ZoomUs.joinMeeting requires meetingNumber')
+  if (typeof meetingNumber !== 'string') meetingNumber = meetingNumber.toString()
+
+  // without noAudio, noVideo fields SDK can stack on joining meeting room for release build
+  return RNZoomUs.joinMeeting({
+    ...params,
+    meetingNumber,
+    noAudio: !!noAudio, // required
+    noVideo: !!noVideo, // required
```

Comment:
would be mo readable to set default values implicitly

Reply:
This way is more reliable. I added defaults too. 
Btw, not sure what default values are used by zoom. I added `false`

Reply:
Ok, I see the defaults above (on line 47), but still I do not see the point of doing `!!`.
With the defaults, if these params are not provided then it will default to boolean.
So it should be simply:
```suggestion
    noAudio,
    noVideo,
```

if you want to guard users of this function to not pass anything else but boolean then on the top of the function there should be a check `typeof noVideo === 'boolean'` etc.

Reply:
I added because want to avoid null. I heard info that it creates bugs for zoom (IOS).

Reply:
That's ok for now. However, I like to be strict. This will accept values like `string` - throwing at the beginning of function would force developer to use boolean.
Also putting defaults/guards/(possibly converting) at the beginning of the function makes it more readable.